app/src/main/java/com/eveningoutpost/dexdrip/Services/DoNothingService.java

```diff
@@ -114,7 +114,8 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             }
         }
 
-        if (CollectionServiceStarter.isFollower(getApplicationContext())) {
+        if (CollectionServiceStarter.isFollower(getApplicationContext()) ||
```

Comment:
What is the reason for attaching to the sync follower service? Wouldn't this implementation be closer to `NSEmulator` data collection type? I don't think any service is required.

Reply:
Maybe I did not understand it correctly. I thougth , DexCollectionType::getCollectorServiceClass would return a DexCollectionsService anyways for NSEmulator. Therefore I used the DoNothingService instead